committee-steering/governance/sig-governance-template-long.md

```diff
@@ -0,0 +1,449 @@
+it is accepted.
+
+> **Note:** To those forking this template - consider changing the
+  following text to only include the tools actively used by your SIG.
+
+In SIG ~~Foo~~, proposals are made using a commonly used tool
+(issues, PRs, docs, etc) and then published to the SIG
+through kubernetes-sig~~foo~~-members@googlegroups.com mailing list
+with the subject containing either `KEP: ` or `RFC: `.  This provides
+a simple mechanism to subscribe to proposals that is less noisy than
+subscribing to github notifications.
+
+Discussion may occur on the issue / PR / doc itself.  Any comments
+on the doc *by SIG members* that are requesting a change
+are considered blocking until they are responded to at least once.
+In cases, that response maybe some variation of "No".
+
+Once the comments are responded to, they are considered addressed
+unless either there is another response *or* the comment has some
+language to indicate it is a blocking issue that must be explicitly
+resolved.  e.g. *lets hold off on doing this until...*
+
+**Optional:**
+
+Once all comments have been addressed on the proposal without responses,
+another email may be sent to the mailing list with `LastCall: KEP: ` or
+`LastCall: RFC: `.  This is to ensure that everyone involved in
+the decision making process understands the proposal will be
+considered accepted if there are no further comments.  If
+there are no further comments or responses to the
+mail in 1-2 business days the proposal can be considered accepted.
+Decisions that have gone through the last call have a high commitment
+with respect to feedback given after the proposal has been accepted.
+
+**Important:** Once a proposal has been accepted then the decision
+has been made and a commitment has been given to whoever implements the
+proposal that their contribution will be accepted if it is
+well implemented.  If it is deemed necessary to reverse a decision,
+this should be done as a new proposal and require lazy consensus on
+the reversal.
+
+In cases where consensus cannot be achieved in 1-2 weeks, the proposal
+may be escalated to the SIG ~~Foo~~ technical leads.  The leads should
+ensure that they are in agreement on the resolution.
+
+#### Defaulting toward action
+
+In cases where there is consensus that a problem exists, but a
+resolution cannot be achieved through consensus or escalation
+to technical leaders, then the default resolution should be
+that the individuals staffing the effort make the decision.  This
+is to avoid deadlock and stagnation when addressing known issues
+with multiple valid solutions.
+
+### Test health ownership and triaging issues
+
+#### Test dashboard
+
+SIG ~~Foo~~ maintains a dashboard on [testgrid](https://k8s-testgrid.appspot.com/sig-foo-master)
+displaying all of the tests the certify the SIG ~~Foo~~ area as
+healthy and the artifacts are in a releasable state.
+
+#### Oncall
+
+In order to facilitate consistently passing tests and triage of issues,
```

Comment:
Again, I think this is impossible with a volunteer army.  I would much prefer to block a release until a bar is met.

Reply:
>  Again, I think this is impossible with a volunteer army. I would much prefer to block a release until a bar is met.

How do you see this working and who do you think should be accountable for tests passing?  How do you see a volunteer army reliably keeping a codebase healthy without heroics on the part of a few individuals?  I believe Google its own test oncall rotation for multi-cluster.  Are you opposed to opening up rotations such as this one to the community?

We have tried blocking the release in the past and IIRC we gave up since putting that put the full burden of test health across the project on SIG release and SIG testing - which was not sustainable.  Today we don't look at each SIG's tests as part of the release.  An example of this in practice was that for 1.9 SIG cli's tests were broken at the time of the release and we released anyway without anyone noticing until weeks afterward.

FWIW, the short template has a lower overhead process for keeping tests healthy.

Reply:
> Are you opposed to opening up rotations such as this one to the community? 

Not opposed, but folks need to volunteer for it, vs. taking it onto a leadership role as part of a release.  E.g. have a test-release czar/captain for a sig.  It is a non-glorious job that can take a lot of work.